neo4j/_async/driver.py

```diff
@@ -225,19 +229,32 @@ def bookmark_manager(
     ) -> AsyncBookmarkManager:
         """Create a :class:`.AsyncBookmarkManager` with default implementation.
 
-        Basic usage example to configure the driver with the default
-        bookmark manager implementation so that all work is automatically
-        causally chained (i.e., all reads can observe all previous writes
-        even in a clustered setup)::
+        Basic usage example to configure sessions with the builtin bookmark
+        manager implementation so that all work is automatically causally
+        chained (i.e., all reads can observe all previous writes even in a
```

Comment:
causal consistency makes only sense in the context of clusters anyway, or?

Reply:
Yes. Do you think this is worth mentioning here?

Reply:
I'd simply remove the "even" bit. Afaik, consistency models do not apply to single instances (they're supposed to be consistent with themselves else it's just... broken)

Reply:
misread the sentence, sorry